pytorch_lightning/overrides/data_parallel.py

```diff
@@ -156,6 +156,8 @@ class LightningDistributedDataParallel(DistributedDataParallel):
     Override the forward call in lightning so it goes to training and validation step respectively
     """
 
+    prepare_for_backwards = True
```

Comment:
what is the case that this will be changed?
as Class contact shall it be capital to separate that tit will be the same for all instances...